jbpm-wb-kie-server/jbpm-wb-kie-server-backend/src/main/java/org/jbpm/workbench/ks/integration/KieServerIntegration.java

```diff
@@ -271,17 +268,18 @@ protected void buildClientsForServer(ServerTemplate serverTemplate) {
                                                                       null,
                                                                       getCredentialsProvider());
         if (kieServicesClient != null) {
-            serverTemplatesClients.put(serverTemplate.getId(),
-                                       kieServicesClient);
+            serverTemplatesClients.computeIfAbsent(serverTemplate.getId(),
+                                                   (k) -> new ConcurrentHashMap<String, KieServicesClient>());
+            serverTemplatesClients.get(serverTemplate.getId()).put("",
```

Comment:
could we use something more meaningful than empty string? quite difficult to figure out why empty string is used as key....

Reply:
I'm open to suggestions :) At the same time, we don't want to have something that has a chance of colliding with the container id.

Reply:
I'd use something that would help to understand what's the usage of that entry - like ```_MAIN_``` ```_BASE_```  ```_GLOBAL_``` etc

Reply:
done